assets/scss/core/_typography.scss

```diff
@@ -25,7 +25,9 @@ $dp-headers-h4-font-size: 22px;
 $dp-form-font-size: 20px;
 $dp-headers-h5-font-size: 18px;
 $dp-headers-h6-font-size: 13px;
-$dp-font-size-small: 13px;
+$dp-font-size-small: 14px;
```

Comment:
Does this change not affect other pages?

Reply:
Yes, it's talking with Santiago

Reply:
This are the current sizes:
11px(x2),12px,13px(x2),14px(x2),15px(x2),16px,18px(x2),20px,22px(x),24px,30px,40px,50px.

Clearly we have a problem here.

Reply:
Yes, missing better definition of font sizes